src/managers/SessionLockManager.cpp

```diff
@@ -176,3 +177,14 @@ bool CSessionLockManager::isSessionLockPresent() {
 bool CSessionLockManager::anySessionLockSurfacesPresent() {
     return m_pSessionLock && std::ranges::any_of(m_pSessionLock->vSessionLockSurfaces, [](const auto& surf) { return surf->mapped; });
 }
+
+bool CSessionLockManager::shallConsiderLockMissing() {
+    Debug::log(LOG, "Shall lock called"); //Locked since got locked by {:i}", m_pSessionLock->mLockTimer.getSeconds());
+
+    if (!m_pSessionLock)
+        return false;
+
+    static auto LOCKDEAD_SCREEN_DELAY = CConfigValue<Hyprlang::INT>("misc:lockdead_screen_delay");
+
+    return static_cast<int>(m_pSessionLock->mLockTimer.getMillis()) > *LOCKDEAD_SCREEN_DELAY;
```

Comment:
useless cast?

Reply:
I gave the habit to use only explicit cast and I saw getMillis returns float but it is not mandatory. Removed it.